types/bitcoin-computer__lib/index.d.ts

```diff
@@ -345,7 +345,7 @@ declare class Computer {
     sign(transaction: Transaction, opts?: SigOptions): Promise<void>;
     fund(tx: Transaction, opts?: Fee & FundOptions): Promise<void>;
     broadcast(tx: Transaction): Promise<string>;
-    sync(rev: string): Promise<unknown>;
+    sync(rev: string): Promise<any>;
```

Comment:
The docs seem to suggest the original type?  
<https://docs.bitcoincomputer.io/api/sync/#type>